### ISSUE TITLE
Weird Hanging Issue When Viewing Newest Posts Comments

### ISSUE THREAD

**Issue by ajayyy** [ludumdare/ludumdare#640]
When I view the newest posts comments, it shows a loading bar, and it says "Please Sign in to comment".
![image](https://cloud.githubusercontent.com/assets/12688112/25110151/c19e5246-23af-11e7-8ea4-5176e0b64abe.png)

@shockeye on the ldjam website mentioned this aswell.

The comments look great on the other post.

**Comment by povrazor** [comment]
What browser+os? Does this happen all the time or just sometimes?

Also, can you check the JavaScript console for anything suspicious. i.e. push F12, click the Console tab.

**Comment by ajayyy** [comment]
Sorry, forgot to mention. I'm on Chrome Windows 10 (older version before anniversary update if that matters). 

Also, there do seem to be errors:
`GET https://api.ldjam.com/vx/node/get/3671+24+9738+1086+320+39+29+1192+9523+817+3205+9722+11+35+9954+104+12131+8159+3639+1214+26 400 ()
Get @ all.min.js?v=2261-760dbb5:1
Get$2 @ all.min.js?v=2261-760dbb5:1
GetKeyed @ all.min.js?v=2261-760dbb5:1
t.getAuthors @ all.min.js?v=2261-760dbb5:7
(anonymous) @ all.min.js?v=2261-760dbb5:7
all.min.js?v=2261-760dbb5:4 Potentially unhandled rejection [1] TypeError: Cannot read property '26' of null
    at t.renderPostNew (https://ldjam.com/-/all.min.js?v=2261-760dbb5:7:9098)
    at t.render (https://ldjam.com/-/all.min.js?v=2261-760dbb5:7:9966)
    at renderComponent (https://ldjam.com/-/all.min.js?v=2261-760dbb5:1:7715)
    at rerender (https://ldjam.com/-/all.min.js?v=2261-760dbb5:1:1392)
    at F (https://ldjam.com/-/all.min.js?v=2261-760dbb5:4:21662)
    at M (https://ldjam.com/-/all.min.js?v=2261-760dbb5:4:21294)
    at S.when (https://ldjam.com/-/all.min.js?v=2261-760dbb5:4:24965)
    at E.run (https://ldjam.com/-/all.min.js?v=2261-760dbb5:4:26074)
    at e._drain (https://ldjam.com/-/all.min.js?v=2261-760dbb5:4:15922)
    at drain (https://ldjam.com/-/all.min.js?v=2261-760dbb5:4:15576)
(anonymous) @ all.min.js?v=2261-760dbb5:4
r @ all.min.js?v=2261-760dbb5:4
l @ all.min.js?v=2261-760dbb5:4`

**Comment by madbook** [comment]
Same thing here – hangs every time, with a similar error:

```javascript
all.min.js:4 Potentially unhandled rejection [2] TypeError: Cannot read property '6296' of null
    at t.renderPostNew (https://ldjam.com/-/all.min.js?v=2261-760dbb5:7:9098)
    at t.render (https://ldjam.com/-/all.min.js?v=2261-760dbb5:7:9966)
    at renderComponent (https://ldjam.com/-/all.min.js?v=2261-760dbb5:1:7715)
    at rerender (https://ldjam.com/-/all.min.js?v=2261-760dbb5:1:1392)
    at F (https://ldjam.com/-/all.min.js?v=2261-760dbb5:4:21662)
    at M (https://ldjam.com/-/all.min.js?v=2261-760dbb5:4:21294)
    at S.when (https://ldjam.com/-/all.min.js?v=2261-760dbb5:4:24965)
    at E.run (https://ldjam.com/-/all.min.js?v=2261-760dbb5:4:26074)
    at e._drain (https://ldjam.com/-/all.min.js?v=2261-760dbb5:4:15922)
    at drain (https://ldjam.com/-/all.min.js?v=2261-760dbb5:4:15576)
```

**Comment by povrazor** [comment]
Wild. In your case @ajayyy it's dying on a specific author whose data it doesn't seem to have. In your case @madbook it's dying on you. :ghost:

**Comment by madbook** [comment]
noooooo – let me know if there's anything I can do to help debug!

**Comment by ajayyy** [comment]
@povrazor Just to confirm, it seems to be the same number every time for the "read property"

**Comment by povrazor** [comment]
I'm going to fire up my Windows 10 machine, see if that helps me reproduce it.

**Comment by zwrawr** [comment]
Clicking on Ajays link

https://api.ldjam.com/vx/node/get/3671+24+9738+1086+320+39+29+1192+9523+817+3205+9722+11+35+9954+104+12131+8159+3639+1214+26
gives me

status":400,"response":"Bad Request","message":"Too many
nodes","data":{"caller_id":29}}


Which might be the problem.

**Comment by povrazor** [comment]
Yes. That would definitely do it. Why so many nodes... oh wait... I know why. haha

**Comment by povrazor** [comment]
Okay, so as a temporary fix I bumped the maximum number of nodes in a request from 20 to 50. That should fix things for now. Long term that's not going to be enough though.

Okay, I'm going to rebrand this bug.

**Comment by ajayyy** [comment]
Awesome! Thanks a lot!

**Comment by povrazor** [comment]
Thanks for the reports @ajayyy and @madbook :smile:

I'm actually proud of myself this time. I left myself an easy error code to track down. :wink:

**Comment by povrazor** [comment]
Ah shoot. Already.

![crapalre](https://cloud.githubusercontent.com/assets/1570313/25134060/91467fd4-241c-11e7-82ee-34b6f30b4ee9.png)

Bumping it to 250.

**Comment by zwrawr** [comment]
This should of been fixed via pagination, If we only ever load a max of 25 nodes we shouldn't load be able to hit the author limit. (Although i don't think there a max number of authors on a single post so technically this could still happen)
